app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,132 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import url('https://fonts.googleapis.com/css?family=Lobster|Open Sans|');
+
+body {
+  margin-left: 20px;
+  margin-right: 0 auto;
+}
+
+btn.title_link {
```

Comment:
CSS style guidelines state that we want to separate multi-word styles with the `-` rather than the `_`